app/src/test/java/com/invisibleteam/goinvisible/mvvm/edition/adapter/EditItemAdapterTest.java

```diff
@@ -64,6 +65,7 @@ public void whenTagsListIsNotEmpty_ViewHolderIsBinded() {
                 .getValue()
                 .get();
 
+        assertThat(adapter.getTagsList().size(), is(tagList.size()));
```

Comment:
The matcher should be for all boolean flags in those assertion, i made comment in wrong place